client/jetpack-cloud/sections/partner-portal/credit-card-fields/credit-card-submit-button.tsx

```diff
@@ -54,30 +53,53 @@ export default function CreditCardSubmitButton( {
 		return __( 'Please wait…' );
 	}, [ formStatus, activeButtonText, __ ] );
 
+	const { setCardDataError, setFieldValue, setFieldError } = useDispatch( creditCardStore );
+
+	const handleButtonClick = () => {
```

Comment:
We're inlining this function to be able to use the data from the hooks.

Reply:
Nice. This is way neater now.